lib/features/sks-menu/presentation/sks_menu_screen.dart

```diff
@@ -52,6 +52,8 @@ class _SksMenuView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    debugPrint("_SksMenuView");
```

Comment:
it's good you've used debugPrint instead of print, but afaik we don't really need it, so it can be deleted